suite-native/coin-enabling/src/components/NetworkSymbolSwitchItem.tsx

```diff
@@ -0,0 +1,116 @@
+import { View } from 'react-native';
+import { useDispatch } from 'react-redux';
+
+import { CryptoIcon } from '@suite-common/icons';
+import { networks, NetworkSymbol } from '@suite-common/wallet-config';
+import { Card, HStack, VStack, Text, Switch } from '@suite-native/atoms';
+import { toggleEnabledDiscoveryNetworkSymbol } from '@suite-native/discovery';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { useToast } from '@suite-native/toasts';
+import { Translation } from '@suite-native/intl';
+import { useAlert } from '@suite-native/alerts';
+
+import { useCoinEnabling } from '../hooks/useCoinEnabling';
+
+type NetworkSymbolProps = {
+    networkSymbol: NetworkSymbol;
+    isEnabled: boolean;
+    showToasts: boolean;
+};
+
+const wrapperStyle = prepareNativeStyle<{ isEnabled: boolean }>((utils, { isEnabled }) => ({
+    gap: 12,
+    flexDirection: 'row',
+    paddingVertical: 6,
+    paddingHorizontal: 14,
+    alignItems: 'center',
+    extend: [
+        {
+            condition: !isEnabled,
+            style: {
+                borderColor: utils.colors.borderElevation0,
+                backgroundColor: utils.colors.backgroundTertiaryDefaultOnElevation0,
+                shadowColor: 'transparent',
+            },
+        },
+    ],
+}));
+
+const iconWrapperStyle = prepareNativeStyle(utils => ({
+    paddingVertical: utils.spacings.small,
+}));
+
+export const NetworkSymbolSwitchItem = ({
+    networkSymbol,
+    isEnabled,
+    showToasts,
+}: NetworkSymbolProps) => {
+    const dispatch = useDispatch();
+    const { applyStyle } = useNativeStyles();
+    const { showToast } = useToast();
+    const { enabledNetworkSymbols } = useCoinEnabling();
+    const { showAlert } = useAlert();
+    const { name } = networks[networkSymbol];
+
+    const showOneNetworkSymbolAlert = () =>
+        showAlert({
+            title: <Translation id="moduleSettings.coinEnabling.oneNetworkSymbolAlert.title" />,
+            description: (
+                <Translation id="moduleSettings.coinEnabling.oneNetworkSymbolAlert.description" />
+            ),
+            primaryButtonTitle: (
+                <Translation id="moduleSettings.coinEnabling.oneNetworkSymbolAlert.action" />
+            ),
+            primaryButtonVariant: 'redBold',
+        });
+
+    const handleEnabledChange = (value: boolean) => {
```

Comment:
nit:
```suggestion
    const handleEnabledChange = (isEnabled: boolean) => {
```

Reply:
conflicting with isEnabled on line 45. That is the reason for value

Reply:
I would prefer to alway start boolean values with `is`/`has` or other descriptive prefix. But not gonna hold this PR because of this.

Reply:
Good point, I am changing to `isChecked` to be more descriptive and not to conflict with `isEnabled`

9254a5da9c